lib/Drush/Commands/core/BrowseCommands.php

```diff
@@ -30,7 +35,19 @@ public function browse($path = '', $options = ['browser' => NULL]) {
     $alias = drush_get_context('DRUSH_TARGET_SITE_ALIAS');
     if (drush_sitealias_is_remote_site($alias)) {
       $site_record = drush_sitealias_get_record($alias);
-      $return = drush_invoke_process($site_record, 'browse', [$path], drush_redispatch_get_options(), array('integrate' => TRUE));
+
+      // Experiment 1a.
+      $ret = $this->taskExec('drush browse')->run();
+      // Experiment 1b.
+      $container = \Drush::getContainer();
+      /** @var Application $app */
+      $app = $container->get('application');
+      $command = $app->get('browse');
+      $ret = $this->taskExec($command)->site($site_record)->run();
```

Comment:
This results in `Consolidation\AnnotatedCommand\AnnotatedCommand does not implement CommandInterface, so can't be passed into this task `. How to fix?

Also, note that I have invented a site() method. Thats how I envision a dev indicating that the task should be run on a separate site. Any thoughts on how to implement that?